app/assets/javascripts/autocise_textarea.js

```diff
@@ -0,0 +1,53 @@
+// Get the max height and apply
+// to all textareas when page loads/reloads
+
+$(document).ready(function() {
+  var numberArray = [];
+  $('textarea').each(function(textarea) {
+    $(this).height( $(this)[0].scrollHeight );
+    numberArray.push($(this).innerHeight());
+    var max = Math.max(...numberArray);
+    $(this).innerHeight(max);
```

Comment:
It may not need to change, but I'm not sure I understand this. You set `height` on line 7 and `innerHeight` on this line (10). Why is that?

Reply:
Had to use .Height to calculate the actual height and innerHeight to set the height.